docs/configuring-playbook-bot-maubot.md

```diff
@@ -48,8 +48,16 @@ You should start in the following order
 3. **Create an instance:** An instance is the actual bot. You have to specify a client which the bot instance will use 
 and the plugin (how the bot will behave)
 
+### Server service management
+If you need to do stop and start manually of the Maubot service, you can use systemd command `systemctl` or `just`. 
+
+- **systemctl** : You will need to ssh insto the server and you can then issue `systemctl restart matrix-bot-maubot` as root.  
+
+- **just** : You can run the command from where the ansible playbook is run(probably from your laptop). `just stop-group bot-maubot` and `just start-group bot-maubot`.
+
+
```

Comment:
Why have these generic help messages here (on this page, specifically)? Does this bot need to be frequently restated for some reason?

Reply:
I have just added on a whim because I have been in need of it personally. Sorry.  Such generic message should be on higher level. I can remove it from here.